ui/src/message_popup/renderers/audio_response_summary.jsx

```diff
@@ -0,0 +1,94 @@
+/* @flow weak */
+import _ from 'lodash';
+import React from 'react';
+import VelocityTransitionGroup from "velocity-react/velocity-transition-group";
+import Divider from 'material-ui/Divider';
+
+import ReadMore from './read_more.jsx';
+
+/*
+Component that displays the summary of audio responses.
+*/
+export default React.createClass({
+  displayName: 'AudioResponseSummary',
+
+  propTypes: {
+    gameSession: React.PropTypes.object.isRequired,
+  },
+
+  render() {
+    const audioResponses = [];
+    const reviewedQuestions = {};  // Keeps track of reviewedQuestions to ensure we only show the last response if the user does multiple retries.
```

Comment:
Yes there's something more happening. I was trying to ensure we show only the last response. I've actually modified this to show all audio responses for cases where the user does more than one audio recording.